pkgs/top-level/python-packages.nix

```diff
@@ -10776,6 +10774,8 @@ in modules // {
     };
   };
 
+  gdal = pkgs.gdal.override { pythonPackages = self; };
```

Comment:
is gdal a Python package? Perhaps use a let expression instead? Would it be useful to make it available via django's passthru?